packages/library/src/rpc-request-coalescer.ts

```diff
@@ -29,13 +29,28 @@ export function getRpcTransportWithRequestCoalescing(
         }
         if (coalescedRequestsByDeduplicationKey[deduplicationKey] == null) {
             const abortController = new AbortController();
+            const responsePromise = (async () => {
+                try {
+                    return await transport<TResponse>({
+                        ...config,
+                        signal: abortController.signal,
+                    });
+                } catch (e) {
+                    if (e && typeof e === 'object' && 'name' in e && e.name === 'AbortError') {
```

Comment:
I think it would be nice to have this condition in a helper method like `isAbortError(e)` because catching them must be quite common.

Just double checking: Is this name condition true for all environments?

Reply:
It is!

I'll make this a helper in a future PR. I think I do this in a few places already.